src/index.js

```diff
@@ -152,6 +155,22 @@ const main = async () => {
   logseq.Editor.registerBlockContextMenuItem("❌ Stop float", async (e) => {
     stopFloat(e);
   });
+  // disable floating block
+  // this could be a toggle, if it knew how to find the only (or first)
+  // video on the page (hint, hint)
+  logseq.App.registerCommandPalette(
+    {
+      key: `logseq-helium-plugin-off`,
+      label: `Disable floating (video) block`,
+      keybinding: {
+        mode: 'global',
+        binding: 'mod+d',
+      },
+    },
+    async () => {
+      await stopFloat(stopFloat(block_uuid_start));
+    }
+  );
```

Comment:
to toggle start/stop float, maybe it could look something like this: https://github.com/vyleung/logseq-time-tracker-plugin/blob/6bd799fdd0c1a0281e2c40316fe7191d713b344b/src/index.js#L1290

Reply:
Much better, I'll retract my plugin for now